css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - flex-wrap</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 5px;
+        padding: 5px;
+        justify-content: space-evenly;
+      }
+      .card-container {
+        padding: 5px;
+        flex-basis: calc(100% / 7);
```

Comment:
This is one-seventh, not one-sixth:

---

![image](https://user-images.githubusercontent.com/16143771/144342332-d96b953c-d804-4818-a8c2-002915a97601.png)

Reply:
hey Tim, so originally I did 1/6th, but six containers didnt fit on the page. 
Should I just do that anyway?